client/src/components/Map/markers/othersLocation.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { PersonCircle } from 'react-bootstrap-icons';
+import UserInfo from '../infoWindow/currLocation.js'
+
+class otherLocation extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { showInfo: false };
+        
+    }
+
+
+    render() {
+        return (
+            <div>
+                <PersonCircle color="red" size={50} />
+                {this.state.showInfo && <UserInfo></UserInfo>}
+            </div>
+        )
+    }
+}
+
+export default otherLocation;
```

Comment:
I feel this is a duplicate code from `markers/currLocation`. I think we can just pass down the information to that file, `isUser` can be used to decide what to render in case it's myself or other user

Reply:
Gotcha! Updated it accordingly